assets/css/main.css

```diff
@@ -167,6 +167,11 @@ img {
 /* Nightmare glyph */
 .nightmare, .archwing, .fissureGlyph {
   margin-top: -3px;
+}
+
+.nightmare:not([src*="/svg/"], 
```

Comment:
I think we should keep it simple, just invert image colors so png matches svg.

Reply:
like make all the pngs black?

Reply:
i'll do a pass tonight when i get home